### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by ashkanent**
I did some clean-up as well as a change in our process to post all sprints of our last quarter into the sheet.

### CODE REVIEW

File: src/main/java/com/ashkan/jira/ClientMain.java [hunk 1]
```diff
@@ -1,38 +1,59 @@
...
+					updater.updateSprintReport(sprintReport);
+				} catch (GeneralSecurityException | IOException e) {
+					e.printStackTrace();
+				}
+			});
 		}
 	}
 
+	private void setProjectMapValues() {
+		projectToListOfSprintIds.put("OREF", new ArrayList<Long>(Arrays.asList(907L, 921L, 922L, 923L, 924L, 925L)));
```
Comment (MustafaHaddara) [inline src/main/java/com/ashkan/jira/ClientMain.java:56]: [nit] I don't think you need `new ArrayList<Long>` here, you could use `Arrays.asList()` on its own...

```suggestion
		projectToListOfSprintIds.put("OREF", Arrays.asList(907L, 921L, 922L, 923L, 924L, 925L));
```

File: src/main/java/com/ashkan/jira/auth/OAuthClient.java [hunk 2]
```diff
@@ -21,13 +21,15 @@
...
 		String requestToken;
 
-		try {
-			requestToken = jiraOAuthClient.getAndAuthorizeTemporaryToken();
-			verificationCode = jiraOAuthClient.getVerificationCodeFromUser();
-			accessToken = jiraOAuthClient.getAccessToken(requestToken, verificationCode);
-		} catch (Exception ex) {
-			ex.printStackTrace();
-			return Optional.of(ex);
+		if (verificationCode == null) {
```
Comment (MustafaHaddara) [inline src/main/java/com/ashkan/jira/auth/OAuthClient.java:24]: 👍

File: src/main/java/com/ashkan/jira/sheets/SheetsUpdater.java [hunk 3]
```diff
@@ -14,19 +15,14 @@
...
-import static com.ashkan.jira.sheets.HeaderDetector.HEADER_INFLATED;
-import static com.ashkan.jira.sheets.HeaderDetector.HEADER_INJECTED;
-import static com.ashkan.jira.sheets.HeaderDetector.HEADER_REMOVED;
+import static com.ashkan.jira.sheets.HeaderDetector.*;
 
 @RequiredArgsConstructor
 public class SheetsUpdater {
 
-	private static final String RANGE = "Sheet1!A:V";
+	private static final String RANGE = "OREF!A:M";
```
Comment (MustafaHaddara) [inline src/main/java/com/ashkan/jira/sheets/SheetsUpdater.java:23]: [optional] this should just be the orange so we can pass the project name in

Comment (ashkanent) [reply src/main/java/com/ashkan/jira/sheets/SheetsUpdater.java:23]: Can you explain... do you mean passing the sheet name from `ClientMain` and attaching it here to make the range?

Comment (MustafaHaddara) [reply src/main/java/com/ashkan/jira/sheets/SheetsUpdater.java:23]: Yeah...I think `updateSprintReport()` should eventually accept a `String projectName` parameter somehow. Maybe the `SprintReport` object should have a `projectName` property? 

I'm thinking ahead to when we want to reuse this tool across multiple sheets.

**Pull request merged**